startup.sh

```diff
@@ -1,2 +1,3 @@
 crontab ./crontab && crond
+npm run migration:sync
```

Comment:
I don't think this will work on Windows (`:` is used to open file streams). I might be wrong though - Nobody runs shell scripts on a vanilla cmd anyways...

Reply:
```suggestion
npm run migration-sync
```